### DISCUSSION TITLE
Welcome to ElmahCore Discussions!

### DISCUSSION THREAD

**Discussion by ElmahCore** [ElmahCore/ElmahCore#166] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rwalshAMNH** [comment]
Hi I see filters and search were added in v2.2 however the nuget package is only available at v2.1
